### PR TITLE
Add CMS preview for landing page

### LINKED ISSUE #217: Display a nice home preview when editing home
Currently when editing home it looks like this:
Selection_275

I want it to look as much as possible as the actual outcome. Also add a language switch at the top to switch between then English and the German version and do not render both at the same time.

### LINKED ISSUE #221: Improve imprint preview in CMS
Currently we render the imprint in German and English, German below English. I want only one language version to be rendered at a time and a language switch at the top. See #217

### PR DESCRIPTION
**Pull request by Bakkhos**
Resolve #217  
Resolve #221
Related to #220

Also adds the language switcher to all existing previews (imprint, faq, homepage)

### CODE REVIEW

**Comment by Levino** [comment]
Please fix #217 instead. Closing this PR. Feel free to reopen.

File: src/cms/preview-templates/Preview.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+const LocalePicker: FunctionComponent<{
+  currentLocale: Locale
+  setLocale: Dispatch<SetStateAction<Locale>>
+}> = ({ currentLocale, setLocale }) => (
+  <ul className="nav nav-tabs sticky-top border-0">
+    {R.map(
+      (locale: Locale) => (
+        <LocaleTab
+          locale={locale}
```
Comment (Bakkhos) [inline src/cms/preview-templates/Preview.tsx]:
```suggestion
          locale={locale}
          key={locale}
```

File: src/cms/preview-templates/FAQPreview.tsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
-import React, { FunctionComponent } from 'react'
+import React, { FunctionComponent, useState } from 'react'
```
Comment (Bakkhos) [inline src/cms/preview-templates/FAQPreview.tsx]:
```suggestion
import React, { FunctionComponent } from 'react'
```

File: src/cms/preview-templates/Preview.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  </ul>
+)
+
+const LocaleTab: FunctionComponent<{
+  locale: Locale
+  active: boolean
+  onClick: () => void
+}> = ({ locale, active, onClick }) => (
+  <li className="nav-item">
+    <button className={`nav-link ${active ? 'active' : ''}`} onClick={onClick}>
```
Comment (kiliw) [inline src/cms/preview-templates/Preview.tsx]: Just for convenience we could disable the active locale button so there is no pointer for on hovering with the mouse.

```suggestion
    <button className={`nav-link ${active ? 'active' : ''}`} disabled={active} onClick={onClick}>
```

**Pull request merged**